CHANGELOG.md

```diff
@@ -1,3 +1,350 @@
+- openebs-operator: set webhook policy to Ignore ([#1172](https://github.com/kinvolk/lokomotive/pull/1172)). This works around a bug in Kubernetes, see this [comment](https://github.com/openebs/openebs/issues/3046#issuecomment-642558701) for details.
+- Don't run control plane components as DaemonSets on single control plane node clusters ([#1193](https://github.com/kinvolk/lokomotive/pull/1193)).
+- calico-host-protection: Add custom locked down PSP configuration ([#1274](https://github.com/kinvolk/lokomotive/pull/1274)).
+
+#### Features
+
+- Add Packet CCM to Packet platform ([#1155](https://github.com/kinvolk/lokomotive/pull/1155)).
+- contour: parameterize Envoy scraping interval ([#1229](https://github.com/kinvolk/lokomotive/pull/1229)).
+- Expose --conntrack-max-per-core kube-proxy flag ([#1187](https://github.com/kinvolk/lokomotive/pull/1187)).
+- Add `require_volume_annotation` for restic plugin ([#1132](https://github.com/kinvolk/lokomotive/pull/1132)). This PR is based on top of #1131 and would need to be merged first.
+- Print bootkube journal if cluster bootstrap fails ([#1166](https://github.com/kinvolk/lokomotive/pull/1166)). This makes easier to debug problems with cluster bringup.
+- aws-ebs-csi-driver: Add dynamic provisioning, resizing, snapshot options ([#1277](https://github.com/kinvolk/lokomotive/pull/1277)). Now the user has ability to control the AWS EBS driver to enable or disable provisioning, resizing, snapshotting.
+
+## Documentation
+
+- Add OpenEBS upgrade guide ([#1163](https://github.com/kinvolk/lokomotive/pull/1163)).
+- Add Rook Ceph upgrade guide ([#1165](https://github.com/kinvolk/lokomotive/pull/1165)),
+
+#### Others
+
+- Pull control plane images from Quay ([#1226](https://github.com/kinvolk/lokomotive/pull/1226)). To avoid hitting Docker Hub pulling limits.
+- Bootkube now waits for all controlplane charts to converge before exiting, which should make bootstrapping process more stable ([#1085](https://github.com/kinvolk/lokomotive/pull/1085)).
+
+### Upgrading from v0.5.0
+
+#### Configuration syntax changes
+
+##### AWS
+
+The undocumented `cluster.os_name` parameter was removed since only Flatcar
+Container Linux is supported.
+
+##### Bare-metal
+
+The `cluster.os_channel` parameter got simplified by removing the `flatcar-` prefix.
+
+###### Old:
+
+```hcl
+os_channel = "flatcar-stable"
+```
+
+###### New
+
+```hcl
+os_channel = "stable"
+```
+
+##### Velero
+
+Velero requires an explicit `provider` field to select the provider.
+Example:
+
+```hcl
+component `velero` {
+  provider = "openebs"
+
+  openebs {
+    ...
+  }
+}
+```
+
+#### Cluster upgrade steps
+
+Updating multiple versions at a time is not supported so, if your cluster is older, upgrade to `v0.5.0` and only then proceed with the upgrade to `v0.6.0`.
+
+Some manual steps need to be performed when updating. In your cluster configuration directory, follow these steps:
```

Comment:
```suggestion
Some manual steps need to be performed when updating. In your cluster configuration directory, follow the steps below.
```

I find "follow these steps" with the colon confusing given that we have just one step (exporting the assets dir var) before the next heading.